config/jobs/kubernetes-sigs/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -183,7 +183,7 @@ presubmits:
             - name: CONTROL_PLANE_MACHINE_COUNT
               value: "3"
             - name: KUBERNETES_VERSION
-              value: 1.23.5
+              value: "latest-1.24"
```

Comment:
should this just be `"latest"` if the intent is to test k8s at HEAD with cloud-provider-azure PRs against the main branch?

Reply:
@lzhecheng @feiskyer what is the intent here? Do we want to test the latest CI bits possible with this job? Or do we want to test the latest bits on an active release branch?

For example:

- v1.25.0-alpha.0.597+aa49dffc7f24dc (latest bits, the result of `value: "latest"`)

or

- v1.24.1-rc.0.29+1868a3a2eb9945 (latest releasable bits, the result of `value: "latest-1.24"`)

Reply:
After thinking about this a little longer it does seem like `"latest"` is the obvious correct choice here. I've made that change.

Reply:
agreed, this should be "latest"